Sources/OpenFeature/Value.swift

```diff
@@ -29,46 +29,21 @@ public enum Value: Equatable {
         }
     }
 
-    // swiftlint:disable:next cyclomatic_complexity
     public func getTyped<T>() -> T? {
         if let value = self as? T {
             return value
         }
```

Comment:
Do we still need one, despite the switch/case logic below?

Reply:
I wasn't really sure what this was for - but I assumed it was needed for something non-obvious.